src/initServiceWorker.ts

```diff
@@ -0,0 +1,19 @@
+if ("serviceWorker" in navigator) {
+  navigator.serviceWorker.register(new URL("./sw.js", import.meta.url)).then(
+    function (registration) {
+      console.log("COOP/COEP Service Worker registered", registration.scope);
+    },
+    function (err) {
+      console.log("COOP/COEP Service Worker failed to register", err);
+    }
+  );
+
+  navigator.serviceWorker.getRegistration().then(function (reg) {
+    // TODO: Or use some other solution
+    // https://stackoverflow.com/questions/51597231/register-service-worker-after-hard-refresh
+    // https://github.com/mswjs/msw/issues/98#issuecomment-612118211
+    if (reg !== undefined && reg.active && !navigator.serviceWorker.controller) {
+      window.location.reload();
+    }
```

Comment:
This doesn't seem to work, as in, the site doesn't get reloaded. I should probably use one of the other approaches anyways

Reply:
Seems to work now